tools/.eslintrc.json

```diff
@@ -105,6 +105,12 @@
                     }
                 ]
             }
+        },
+        {
+            "files": ["./base/src/version.js"],
+            "rules": {
+                "notice/notice": "off"
+            }
```

Comment:
Why was this needed?

Reply:
If the generator can't add this information, should we add this file to `.eslintignore` instead?

Reply:
Or I guess the question is why isn't listing it there _enough_?

Reply:
I think the intension here is to allow the linter to run on both the files (d.ts & .js) as `tools/**/*.d.ts
` excludes `tools/base/src/version.d.ts` file from tools in .eslintignore 
Then to ignore the header error for version.js we had to add the above override rule inside .eslintrc.json.

I believe we can safely remove the last line `!tools/base/src/version.js` as it's included by default. Although if we just add `tools/base/src/version.js` to .eslintignore we would not require the above change in .eslintrc.json. 

@rickharris why is it necessary to add version.d.ts & version.js files to linter ?

Reply:
Yeah I think that we can eslintignore the files if we'd like to. I also think you're right that we can remove `!tools/base/src/version.js` from the eslintignore file. Either option should be fine, just a preference of if we want to ignore the whole file or just the header rule.

Reply:
@Westbrook  added tools/base/src/version.js to .eslintignore & removed the header override.

Reply:
Are we able to remove the updates to this file with that change?

Reply:
Yes Removed it.